CI/travis.osx.install.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+if [ "${TRAVIS_EVENT_TYPE}" == "cron" ] \
+  && [ "${CC}" != "clang" ] && [ "${Q_OR_C_MAKE}" != "qmake" ]; then
```

Comment:
We have only clang on macOS, so the condition is unnecessary.

Reply:
Fixed.